var/spack/repos/builtin/packages/glm/package.py

```diff
@@ -14,25 +14,33 @@ class Glm(CMakePackage):
     homepage = "https://github.com/g-truc/glm"
     url = "https://github.com/g-truc/glm/archive/0.9.9.8.tar.gz"
 
+    version('develop', git="https://github.com/g-truc/glm.git")
     version('0.9.9.8', sha256='7d508ab72cb5d43227a3711420f06ff99b0a0cb63ee2f93631b162bfe1fe9592')
     version('0.9.7.1', sha256='285a0dc8f762b4e523c8710fbd97accaace0c61f45bc8be2bdb0deed07b0e6f3')
 
     depends_on('cmake@2.6:', type='build')
     depends_on('cmake@3.2:', type='build', when='@0.9.9.0:')
+    depends_on('cmake@3.2:', type='build', when='@develop')
```

Comment:
This is redundant, since `@develop` should be greater of any other "regular" version

Reply:
Thanks @alalazo.  I assumed numbers and text were not comparable.  Is "develop" a special word that always compares greater than a numbered version or is that true of all pure character versions?

Reply:
Short answer is it's a special word, long answer [here](https://spack.readthedocs.io/en/latest/packaging_guide.html#version-comparison) 🙂